src/components/Reports/ReportsNew.test.js

```diff
@@ -50,4 +51,69 @@ describe('Reports New page', () => {
     // Assert
     await waitFor(() => expect(getByText('common.unexpected_error')));
   });
+
+  if (
```

Comment:
Why is this formatted in such a weird way? prettier?

Reply:
```suggestion
  it (
```

Reply:
yes i changed!